SU2_CFD/src/variables/CNEMONSVariable.cpp

```diff
@@ -78,7 +78,7 @@ CNEMONSVariable::CNEMONSVariable(su2double val_pressure,
 
 bool CNEMONSVariable::SetPrimVar(unsigned long iPoint, CFluidModel *FluidModel) {
 
-  unsigned long iVar, iSpecies;
+
 
```

Comment:
```suggestion
```

Reply:
Important change, I know :D

Reply:
Perfect! Hahah ill change this.  The main motivation for doing this is because I keep running into inconsistencies within branches!

Reply:
clang-format is your friend...